src/aleph/http.clj

```diff
@@ -29,22 +29,25 @@
+   | `socket-address`                  |  a `java.net.SocketAddress` specifying both the port and interface to bind to.
+   | `bootstrap-transform`             | a function that takes an `io.netty.bootstrap.ServerBootstrap` object, which represents the server, and modifies it.
+   | `ssl-context`                     | an `io.netty.handler.ssl.SslContext` object or a map of SSL context options (see `aleph.netty/ssl-server-context` for more details) if an SSL connection is desired |
+   | `manual-ssl?`                     | set to `true` to indicate that SSL is active, but the caller is managing it (this implies `:ssl-context` is nil). For example, this can be used if you want to use configure SNI (perhaps in `:pipeline-transform`) to select the SSL context based on the client's indicated host name. |
+   | `pipeline-transform`              | a function that takes an `io.netty.channel.ChannelPipeline` object, which represents a connection, and modifies it.
+   | `executor`                        | a `java.util.concurrent.Executor` which is used to handle individual requests.  To avoid this indirection you may specify `:none`, but in this case extreme care must be taken to avoid blocking operations on the handler's thread.
+   | `shutdown-executor?`              | if `true`, the executor will be shut down when `.close()` is called on the server, defaults to `true`.
+   | `request-buffer-size`             | the maximum body size, in bytes, which the server will allow to accumulate before invoking the handler, defaults to `16384`.  This does *not* represent the maximum size request the server can handle (which is unbounded), and is only a means of maximizing performance.
+   | `raw-stream?`                     | if `true`, bodies of requests will not be buffered at all, and will be represented as Manifold streams of `io.netty.buffer.ByteBuf` objects rather than as an `InputStream`.  This will minimize copying, but means that care must be taken with Netty's buffer reference counting.  Only recommended for advanced users.
+   | `rejected-handler`                | a spillover request-handler which is invoked when the executor's queue is full, and the request cannot be processed.  Defaults to a `503` response.
+   | `max-initial-line-length`         | the maximum characters that can be in the initial line of the request, defaults to `8192`
+   | `max-header-size`                 | the maximum characters that can be in a single header entry of a request, defaults to `8192`
+   | `max-chunk-size`                  | the maximum characters that can be in a single chunk of a streamed request, defaults to `16384`
+   | `validate-headers`                | if `true`, validates the headers when decoding the request, defaults to `false`
+   | `initial-buffer-size`             | configures the initial buffer size when decoding the request, defaults to `128`
```

Comment:
Maybe say what it's measured in? (Bytes? Kbs? Pages? Objects?)

Reply:
Sure!
It's `bytes` obviously, I will also add for the other ones.

Reply:
It's a buffer size of characters.